dotcom-rendering/src/server/htmlPageTemplate.ts

```diff
@@ -363,14 +363,19 @@ https://workforus.theguardian.com/careers/product-engineering/
 					renderingTarget === 'Web'
 						? `
                 <noscript>
+					<!-- Comscore Identifier: comscorekw=${props.keywords
+						.replace(/ /g, '_')
+						.replace(/&/g, 'and')} -->
```

Comment:
regex footguns give me the fear. Could we use [`replaceAll`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll) instead?

Reply:
Yep good shout - I'll update the commercial PR to do the same